### DISCUSSION TITLE
Some problems with result

### DISCUSSION THREAD

**Discussion by eSpadaMaddyS** [kyleskom/NBA-Machine-Learning-Sports-Betting#51] [Q&A] [answered]
Hi guys!Now i trynna run your app:
My configure:
Python==3.8.
requests==2.25.1
tqdm==4.59.0
colorama==0.4.4
pandas==1.5.2
numpy==1.21.0
scikit_learn==1.0
keras==2.6.0
tensorflow==2.6.0
xgboost==1.4.2

My problem:
![image](https://user-images.githubusercontent.com/113463164/209038622-f39ab068-a6ff-4a01-b269-d4efaaca0c62.png)

I entered total of points,odds,but i dont have the result.Mb i do something wrong?

**Comment by kyleskom** [comment]
What is the full command you used to run?

**Comment by eSpadaMaddyS** [comment, marked as answer]
I'm sorry for botherin you.My problem was in IP configuration.I probably lose my connection with nba.com sometimes.Now it works.I hope that your work with automatic odds will be fine.Good luck)
